org.yakindu.examples.raspberry.helloworld/.project

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.yakindu.examples.raspberry.helloworld</name>
+	<comment></comment>
+	<projects>
+		<project>coffee_machine_pro</project>
```

Comment:
Line number 6 was a reference to the project coffee_machine_pro which I just removed. Or did you mean the name tag?